pkgs/top-level/all-packages.nix

```diff
@@ -28044,6 +28044,8 @@ with pkgs;
 
   qxw = callPackage ../applications/editors/qxw {};
 
+  qt-jdenticon = libsForQt5.callPackage ../development/libraries/qt-jdenticon {};
```

Comment:
This is a library hence it should be inside `qt5-packages.nix`.

Reply:
Now it’s added to `qt6-packages.nix` instead of `all-packages.nix`